data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/defense/defense_add_divide.cfg

```diff
@@ -0,0 +1,36 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [defense]value=
+##
+# Actions:
+# Give the leaders a defense ability with both the value, add, and divide attributes, adding up to 100
+# Give each unit 1000 hp and 100 strikes
+# Have the side 1 leader attack the side 2 leader with both its weapons
+##
+# Expected end state:
+# The side 1 leader's first weapon strikes 0 times
+# The side 1 leader's second weapon strikes 0 times
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "defense_add_divide" (
+    [event]
+        name = start
+
+        {SET_HP VALUE=1000}
+
+        [modify_unit]
+            [filter]
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {TEST_ABILITY attacks 100 () SELF=yes}
+                    {TEST_ABILITY defense 50 (add=150
+                    divide=2) SELF=yes}
```

Comment:
So since the value attribute is present, add and divide are applied to the value attribute. If the value attribute wasn't present then they would be calculated using the unit's normal stats, right?